index.js

```diff
@@ -16,3 +16,59 @@ async function start() {
   // Now try and complete the program.
   process.exit();
 }
+fconst readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+start();
+
+async function start() {
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+  console.log("You entered: " + secretNumber);
+  let min = 0;
+  let max = 100;
+  //narrowing range//
+
+  //start project here//
+  //genertaing a random number between 1 to 100 for user//
+  console.log(randomNum(0, 100));
+  let response = await ask("Is this your number?");
```

Comment:
This looks like you're on the right track, some bugs probably stopped you from fleshing it out more but I think you're following the theory and logic 👍